x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/transforms/ClientTransformIndexer.java

```diff
@@ -541,25 +627,31 @@ private boolean isIrrecoverableFailure(Exception e) {
     }
 
     synchronized void handleFailure(Exception e) {
-        logger.warn(new ParameterizedMessage("[{}] transform encountered an exception: ",
-            getJobId()),
-            e);
+        logger.warn(
+            new ParameterizedMessage(
+                "[{}] transform encountered an exception: ",
+                getJobId()
+            ),
+            e
+        );
         if (handleCircuitBreakingException(e)) {
             return;
         }
 
-        if (isIrrecoverableFailure(e) || failureCount.incrementAndGet() > transformTask.getNumFailureRetries()) {
-            String failureMessage = isIrrecoverableFailure(e) ?
-                "task encountered irrecoverable failure: " + e.getMessage() :
-                "task encountered more than " + transformTask.getNumFailureRetries() + " failures; latest failure: " + e.getMessage();
+        if (isIrrecoverableFailure(e) || failureCount.incrementAndGet() > context.getNumFailureRetries()) {
+            String failureMessage = isIrrecoverableFailure(e)
+                ? "task encountered irrecoverable failure: " + e.getMessage()
```

Comment:
if this is how our new formatting is (putting `?` on the next line), I don't like it.